bin/lts-candidate-stats

```diff
@@ -64,9 +86,44 @@ class Script {
     static String ageIndicator(it) {
         def releasedAs = it.customfields.customfield.find { it.customfieldname == "Released As" }.customfieldvalues as String
         if (releasedAs?.trim()) {
-            return releasedAs
+            return releasedAs.replaceAll("[Jj]enkins ", "")
```

Comment:
Would look for `\d[.]\d\d\d($|[^.])` specifically instead. Someone might use a variant like `jenkins-2.229` as the field help text suggests.

Reply:
This is used only for visualizing, and since 99.9% backported issues are those from Jenkins, I saw no point to be explicit about that so it is hidden from the output. Though in case the component would be different, it would actually be desirable to keep the component name there since version alone would not mean much.

Thanks for pointing out the not-much-used `jenkins-2.229` syntax. Adjusting accordingly.

Reply:
Good point, somehow I misread that as only keeping the version number, instead of removing a redundant prefix. So this is fine, just a minor convenience feature that will not falsely hide content.